models/models.go

```diff
@@ -93,6 +93,7 @@ type Record struct {
 	Languages             []Language            `json:"languages,omitempty"`
 	ProviderPerformances  []ProviderPerformance `json:"provider_performances,omitempty"`
 	InsuranceProviders    []InsuranceProvider   `json:"insurance_providers,omitempty"`
+	DiagnosticStudyOrders []Procedure           `json:"diagnostic_study_order,omitempty"`
```

Comment:
The Record model in Health Data Standards doesn't have a specific field for diagnostic study orders since they're treated as Procedures. This means we want the diagnostic study orders to be in the `procedures` section of the JSON that's passed back to Ruby, so I'd probably rewrite this line to just be 
```
Procedures []Procedure  `json:"procedures",omitempty`
```